lbry/testcase.py

```diff
@@ -329,13 +348,18 @@ async def generate(self, blocks):
         """ Ask lbrycrd to generate some blocks and wait until ledger has them. """
         prepare = self.ledger.on_header.where(self.blockchain.is_expected_block)
         height = self.blockchain.block_expected
+        self.conductor.spv_node.server.es_synchronized.clear()
+        self.conductor.spv_node.server.go_hub_synchronized.clear()
         self.conductor.spv_node.server.synchronized.clear()
         await self.blockchain.generate(blocks)
         await prepare  # no guarantee that it didn't happen already, so start waiting from before calling generate
         while True:
             await self.conductor.spv_node.server.synchronized.wait()
+            self.conductor.spv_node.server.es_synchronized.clear()
+            self.conductor.spv_node.server.go_hub_synchronized.clear()
             self.conductor.spv_node.server.synchronized.clear()
-            if self.conductor.spv_node.server.db.db_height >= height:
+            if self.conductor.spv_node.server.db.db_height >= height and \
+                self.conductor.spv_node.server._go_hub_height == self.conductor.spv_node.server.db.db_height == self.conductor.spv_node.server._es_height:
                 break
```

Comment:
can you set `self.conductor.spv_node.server` to a variable somewhere earlier in the function and then use the variable, this would be significantly faster (attribute lookup has a cost in Python) and it would make the code more readable